addons/mail/models/mail_channel.py

```diff
@@ -785,39 +810,71 @@ def channel_fetched(self):
             channel_partner.write({
                 'fetched_message_id': last_message_id,
             })
-            data = {
-                'id': channel_partner.id,
-                'info': 'channel_fetched',
-                'last_message_id': last_message_id,
-                'partner_id': self.env.user.partner_id.id,
-            }
-            self.env['bus.bus'].sendmany([[(self._cr.dbname, 'mail.channel', channel.id), data]])
+            self.env['bus.bus'].sendmany([[
+                channel,
+                'mail.channel_fetched',
+                 {
+                    'id': channel_partner.id,
+                    'last_message_id': last_message_id,
+                    'partner_id': self.env.user.partner_id.id,
+                 }
+            ]])
 
     def channel_set_custom_name(self, name):
         self.ensure_one()
         channel_partner = self.env['mail.channel.partner'].search([('partner_id', '=', self.env.user.partner_id.id), ('channel_id', '=', self.id)])
         channel_partner.write({'custom_channel_name': name})
-        self._broadcast(self.env.user.partner_id.ids)
+        notifications = []
+        for partner in self.channel_partner_ids:
+            notifications.append([
+                partner,
+                'mail.custom_name',
+                {
+                    'id': self.id,
+                    'name': name,
+                }
+            ])
+        self.env['bus.bus'].sendmany(notifications)
 
     def channel_rename(self, name):
         self.ensure_one()
         self.write({'name': name})
-        self._broadcast(self.channel_partner_ids.ids)
+        notifications = []
+        for partner in self.channel_partner_ids:
+            notifications.append([
+                partner,
+                'mail.rename',
+                {
+                    'id': self.id,
+                    'name': name,
+                }
+            ])
```

Comment:
guideline: single line

this can be sent on the whole channel, not to each member individually